src/commands/prune.rs

```diff
@@ -589,9 +589,9 @@ impl Pruner {
         };
 
         self.repack_candidates.sort_unstable_by_key(|rc| rc.0);
-        let mut resize_packs: BlobTypeMap<Vec<_>> = Default::default();
-        let mut do_repack: BlobTypeMap<bool> = Default::default();
-        let mut repack_size: BlobTypeMap<u64> = Default::default();
+        let mut resize_packs: BlobTypeMap<Vec<_>> = BlobTypeMap::default();
```

Comment:
Is there a possibility to simply the type hint now that we are using `BlobTypeMap::default`?